docs/contributing.md

```diff
@@ -64,6 +64,14 @@ We're writing documentation as we are developing new features. If you find
 something that is confusing or not covered at all, feel free to
 [open a bug][ob] or [contribute][cd]
 
+## Contributing to the Design
+
+Since Radicle is an open source project, anyone can contribute. This is normal in open source development, but we do it for design too! Here's how you can get started:
```

Comment:
Could you hard-wrap the lines around 80 characters?

Reply:
Is there a VS Code plugin for this?